torch/_dynamo/variables/builder.py

```diff
@@ -1539,6 +1539,11 @@ def _clone_input(value):
     ]:
         proxy.node.meta["example_value"] = example_value
         return ConstantVariable.create(example_value, **options)
+    elif isinstance(example_value, torch.backends.cuda.SDPAParams):
+        from .flash_attention import SDPAParamsVariable
+
+        proxy.node.meta["example_value"] = example_value
+        return SDPAParamsVariable(proxy, **options)
```

Comment:
guard?

Reply:
Won't we have installed guards for the arguments already? At this point this is an intermediate

Reply:
Oh, yes, this is under wrap_fx_proxy_cls - guards are not needed here.